### DISCUSSION TITLE
Campus Experts languages support ?

### DISCUSSION THREAD

**Discussion by rahman-O** [community/community#46593] [GitHub Education] [answered]
Is it possible to upload a video to Campus Experts  with a language other than English?

**Comment by jge162** [comment]
sure but not sure if they will understand it. but, up to you to try if you wish.

**Reply by rahman-O** [reply]
Is it useful to add a subtitle tape in English and the language of speech is in Arabic?

**Reply by jge162** [reply]
I think it will be useful.

**Comment by samus-aran** [comment, marked as answer]
Hi folks 👋 

Just to provide some clarity for those applying to Campus Experts that video submissions should be in English. We appreciate the effort involved; however the Campus Expert team are unable to translate non English submissions and thank you for understanding.

**Comment by anni-anni** [comment]
In this scenario you can [translate video](https://wavel.ai/solutions/ai-video-translator/) entirely into English using an AI dubbing model which supports both Arabic and English translations|
This would save you time and efforts, and no need to remake the full video!
